crates/fluvio-types/build.rs

```diff
@@ -0,0 +1,17 @@
+use semver::Version;
+
+fn main() {
+    if let Ok(verpath) = std::fs::canonicalize("../../VERSION") {
+        if verpath.exists() {
+            println!("cargo:rerun-if-changed=../../VERSION");
+        }
+    }
+
+    println!("cargo:rerun-if-changed=build.rs");
+
+    // Parse the Fluvio Platform Version
+    let version = include_str!("../../VERSION").trim();
```

Comment:
Why parsing this again? This won't work since it is public crate

Reply:
I think this is my bad. At the time of [my feedback](https://github.com/infinyon/fluvio/pull/3485#issuecomment-1690070729) I didn't consider that this was a public crate